src/monitoring/aws/redshift/metrics.ts

```diff
@@ -5,7 +5,7 @@ const enum Metrics {
   DatabaseConnections = 'DatabaseConnections',
   PercentageDiskSpaceUsed = 'PercentageDiskSpaceUsed',
   CPUUtilization = 'CPUUtilization',
-  Enabled = 'Enabled',
+  MaintenanceMode = 'MaintenanceMode',
```

Comment:
I think we should add it into a separate change with type as `fix` so that a release is created for the same.

Reply:
done